### PR TITLE
provides feature request 57 functionality

### PR DESCRIPTION
**Pull request by ClaudiaFrank**
see #57

### CODE REVIEW

**Comment by chcg** [comment]
@ClaudiaFrank Could you please update the PR and resolve the conflicts.

**Comment by ClaudiaFrank** [comment]
@chcg - any ideas why this isn't working? Branch has no conflicts anymore but all tests failed.

File: PythonScript/src/NotepadPlusWrapper.cpp [hunk 1]
```diff
@@ -1079,10 +1079,23 @@ void NotepadPlusWrapper::disableAutoUpdate()
 	callNotepad(NPPM_DISABLEAUTOUPDATE, 0, 0);
 }
 
+<<<<<<< HEAD
```
Comment (chcg) [inline PythonScript/src/NotepadPlusWrapper.cpp]: @ClaudiaFrank Merge markers are still contained -> build issue

**Comment by ClaudiaFrank** [comment]
will recreate it

**Pull request closed without merging**